scan/scan_agents.py

```diff
@@ -7,27 +8,25 @@ def __init__(self, llm, topic):
         self.llm = llm
         self.topic = topic
 
-    def dlpfc_agent(self):
+    def dlpfc_agent(self, context=""):
```

Comment:
What if context doesn't get set when this is called? It will result in an empty string which I assume isn't what you want? An empty string is going to make debugging harder if something goes wrong. I am assuming that instead you either want to force sending a value here or set it to `None` instead of an empty string?